features/step_definitions/package_json_steps.coffee

```diff
@@ -4,11 +4,13 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I am in a node project$/, (done) ->
-    fsExtra.outputJson path.join(@tmpDir, 'package.json'), name: '123', done
+  @Given /^I work on a Node.js project$/, (done) ->
```

Comment:
Suggestions:
* `I am working` instead of `I work` to clarify that its in the current context
* `Node.js package` instead `Node.js project` as I relate `package.json` to npm